.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+	"name": "Rust for reth",
+
+	"build": {
+		"dockerfile": "./Dockerfile",
+		"context": "."
+	},
+
+	// Use 'mounts' to make the cargo cache persistent in a Docker Volume.
+	// "mounts": [
+	// 	{
+	// 		"source": "devcontainer-cargo-cache-${devcontainerId}",
+	// 		"target": "/usr/local/cargo",
+	// 		"type": "volume"
+	// 	}
+	// ]
```

Comment:
why not uncomment this?

Reply:
this comes from the default template provided my MS - I personally like the devcontainers to be fully isolated from each other so my default view is that I wouldn't bother - but it's likely true that this would improve performance if you're regularly rebuilding the container from scratch (e.g. a CI/CD environment relying upon dev containers)